components/brave_rewards/browser/rewards_service.h

```diff
@@ -312,6 +312,8 @@ class RewardsService : public KeyedService {
 
   virtual void GetExternalWallet(GetExternalWalletCallback callback) = 0;
 
+  virtual std::string GetExternalWalletType() const = 0;
```

Comment:
Do we need to make this public as calling `GetExternalWallet` returns the type in the data structure